ci/nits.sh

```diff
@@ -18,20 +18,23 @@ declare prints=(
 
 # Parts of the tree that are expected to be print free
 declare print_free_tree=(
-  'core/src'
-  'faucet/src'
-  'ledger/src'
-  'metrics/src'
-  'net-utils/src'
-  'runtime/src'
-  'sdk/bpf/rust/rust-utils'
-  'sdk/src'
```

Comment:
Combined with [this](https://github.com/solana-labs/solana/pull/8090/files#r373991081), currently `sdk/src/macro/lib.rs` is escaping from the nits check! ;)